cmd/truss/main.go

```diff
@@ -240,33 +243,41 @@ func generateCode(cfg *truss.Config, dt deftree.Deftree, sd *svcdef.Svcdef) ([]t
 
 	genDocFiles := gendoc.GenerateDocs(dt)
 
-	genFiles := append(genGokitFiles, genDocFiles...)
+	return combineFiles(genGokitFiles, genDocFiles), nil
+}
+
+// combineFiles takes any numbers of fileSystems and combines them into one.
+func combineFiles(group ...fileSystem) fileSystem {
```

Comment:
Will there ever be a case where there can be a conflict? This function silently overwrites an existing key.